src/olympia/versions/tests/test_views.py

```diff
@@ -179,6 +188,14 @@ def test_download_for_unlisted_addon_owner(self):
         self.assert_served_internally(
             self.client.get(url), False, attachment=True)
 
+        # Even allowed to bypass georestrictions.
```

Comment:
Another drive-by question: where are all of the exclusions tested for in this file defined, in terms of requirements?

Reply:
What do you mean ?

Reply:
I mean, who says that we should be doing this exclusion? Who says that it's okay for an owner in a restricted region to be able to download their add-on? (Or maybe I'm just not understanding what's being tested here)

Reply:
It's not defined anywhere so I made the call myself, mostly to make it easier to implement a bypass for admins & reviewers. I don't think it matters much, and it matches what we allow for disabled, so that makes sense.